addons/mail/static/src/models/rtc.js

```diff
@@ -750,26 +748,26 @@ registerModel({
          * Cleans up a peer by closing all its associated content and the connection.
          *
          * @private
-         * @param {String} token
+         * @param {number} sessionId
          */
-        _removePeer(token) {
-            const rtcSession = this.messaging.models['RtcSession'].findFromIdentifyingData({ id: token });
+        _removePeer(sessionId) {
```

Comment:
Can this use `RtcSession` too?

Reply:
It's currently a bit difficult because it's meant to remove peerConnections, not really `RtcSession`s, so there are places where it's difficult to provide a session. And there could also be situations where the record `RtcSession` doesn't exist but the peerConnection (and some related things like `dataChannels` and some timeouts) may still need to be cleaned.

I think we can do it once all those fields are in the models.

Reply:
Ok, this will be easily be fixed with Task-2734101 that turns `Rtc/_peerConnections` and `Rtc/_dataChannels` into `RtcSession` fields.